### PR TITLE
Draft PPP, fix UG/DG headers

### LINKED ISSUE #190: Re: Font size of headers for DG/UG
"#####" is very small

### LINKED ISSUE #204: Write PPP for chuckiex3

### LINKED ISSUE #227: Rishi's DG review Re: Error handling
image

### PR DESCRIPTION
**Pull request by franceneee**

### CODE REVIEW

**Comment by franceneee** [comment]
**For the DG:**
Added 2 horizontal lines to separate sections.

For the *design and implementation headers*, the following changes were made:
5# (design and implementation headers) => bold text
6# (aspect of design)=> italicised text

For Neil's part on storage, I removed 1# from every header.

If you want to see what the dg will look like, you can see a preview at: https://chuckiex3.github.io/tp/DeveloperGuide.html

File: docs/DeveloperGuide.md [hunk 1]
```diff
@@ -18,35 +18,17 @@
...
-&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;[4.3.1.1. Implementation](#4311-implementation) <br>
 &nbsp;&nbsp;&nbsp;&nbsp;[4.3.2. Select Feature](#432-select-feature) <br>
-&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;[4.3.2.1. Implementation](#4321-implementation) <br>
-&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;[4.3.2.2. Design Considerations](#4322-design-considerations) <br>
 &nbsp;&nbsp;&nbsp;&nbsp;[4.3.3. Tag Feature](#433-tag-feature) <br>
-&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;[4.3.3.1. Implementation](#4331-implementation) <br>
-&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;[4.3.3.2. Design Considerations](#4332-design-considerations) <br>
 &nbsp;&nbsp;&nbsp;&nbsp;[4.3.4. Search Feature](#434-search-feature) <br>
-&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;[4.3.4.1. Implementation](#4341-implementation) <br>
-&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;[4.3.4.2. Design Considerations](#4342-design-considerations) <br>
```
Comment (yAOwzers) [inline docs/DeveloperGuide.md:49]: nice

**Review by yAOwzers — APPROVED** [review]

File: docs/team/chuckiex3.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
+# Mooi Suet Yeng, Francene's Project Portfolio Page
+
+## Project: Zer0Note
+//todo standardise
+
+### Summary of Contributions
+- Created ASCII art for Zer0Note
+- New Feature: Added the ability to add tasks and notebooks/sections/pages
+- New Feature: Added the ability to select a notebook/section/page
```
Comment (Lusi711) [inline docs/team/chuckiex3.md]: A bit too minimal. Maybe you could elaborate more on the Justifications and Highlights?

File: docs/team/chuckiex3.md [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+- Code contributed: [Reposense link](https://nus-cs2113-ay2021s1.github.io/tp-dashboard/#breakdown=true&search=chuckiex3&sort=groupTitle&sortWithin=title&since=2020-09-27&timeframe=commit&mergegroup=&groupSelect=groupByRepos&checkedFileTypes=docs~functional-code~test-code~other&tabOpen=true&tabType=authorship&tabAuthor=chuckiex3&tabRepo=AY2021S1-CS2113T-T12-3%2Ftp%5Bmaster%5D&authorshipIsMergeGroup=false&authorshipFileTypes=docs~functional-code~test-code~other)
+- Project Management:
+  - Managed release `v2.0` on GitHub
+- Enhancement:
+  - Updated the `select` function so that users can select a notebook/section/page from anywhere [#62](https://github.com/AY2021S1-CS2113T-T12-3/tp/pull/62)
```
Comment (Lusi711) [inline docs/team/chuckiex3.md:16]: Move this to after introducing features for flow

Comment (franceneee) [reply docs/team/chuckiex3.md:16]: okay, will do xD

File: docs/team/chuckiex3.md (hunk 2 above)
Comment (franceneee) [reply docs/team/chuckiex3.md]: heheheh.... I will craft something... thanks xD

**Review by longngng — COMMENTED** [review]
Looks good! Thanks for editing the user guide 👍 !

**Review by Lusi711 — APPROVED** [review]

**Review by NeilBaner — DISMISSED** [review]
Clean. Thanks, this is tons better.

**Comment by NeilBaner** [comment]
And I see you're on it already :P

**Review by NeilBaner — APPROVED** [review]
It passes now, good enough for me :)

**Pull request merged**